erpnext/stock/doctype/delivery_note/delivery_note.json

```diff
@@ -1249,13 +1250,19 @@
    "fieldtype": "Time",
    "label": "Deliver Before",
    "read_only": 1
+  },
+  {
+   "fieldname": "order_type",
+   "fieldtype": "Select",
+   "label": "Order Type",
+   "options": "Sales\nMarketing"
```

Comment:
This should actually include all the order types, even the ones we've added as a customization